pulsar-functions/api-java/src/main/java/org/apache/pulsar/functions/api/Context.java

```diff
@@ -86,7 +86,14 @@
      *
      * @return the instance id
      */
-    String getInstanceId();
+    int getInstanceId();
+
+    /**
+     * Get the number of instances that invoke this function.
+     *
+     * @return the number of instances that invoke this function.
+     */
+    int getNumInstances();
```

Comment:
getNumInstances won't work in the context of localrun.

Reply:
how about adding a comment there, saying `getNumInstances` will return -1 if the function is invoked from localrun?

Reply:
Just add a comment stating that this might not be accurate during localrun